tests/test_series.py

```diff
@@ -1237,8 +1237,8 @@ def cond2(x: pd.Series[int]) -> pd.Series[bool]:
     cond3 = pd.Series([False, True, True])
     check(assert_type(s.where(cond3, other=0), "pd.Series[int]"), pd.Series, int)
     
- 
- def test_bitwise_opeator() -> None:
+    
```

Comment:
have extra spaces on line above.

You need to run `poetry run poe test` and check to see if any errors are reported.

Reply:
> have extra spaces on line above.
> 
> You need to run `poetry run poe test` and check to see if any errors are reported.

sir on my system no error were reported even after the update